common/SpotifyHelper.ts

```diff
@@ -25,7 +32,7 @@ export class SpotifyHelper {
                 return response.json()
             })
             .catch((e) => {
-                throw e
+                return e
```

Comment:
i'm returning the error because i can't work/it's far harder to work with a thrown error than a definitive thrown one. if i return it, then i can create some logic depending on the type of error it is. i basically made it so that the get function in the spotify helper class returns the error status code and i've created some logic in many other functions on what to do if that status is 401 (bad access token), if the status isn't 401 but still an error (output an error notification or info depending).  so basically i'm saying that if there is an error in fetching some spotify data, i want to know about it rather than just throwing the error and moving on.

Reply:
its throw here so you can put a `.catch(e => //this is the error you want)` where you're using the `get`, and then put the logic inside the catch for whatever part of the error you wanna see.

get is async so when you use it, it should behave like
```
get()
  .then(data => // get was sucecssful)
  .catch(error => //unsuccessful, heres the error)
```